index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Write and read books</title>
     <!-- styles -->
+    <link rel="stylesheet" href="static/css/general.css">

```

Comment:
I think css names can be enhanced based on standards or purpose.
Research about how to improve css structure and naming